pages/linux/hexdump.md

```diff
@@ -0,0 +1,24 @@
+# hexdump
+
+> Display file contents in hexadecimal, decimal, octal, or ascii.
+> More information: <https://man7.org/linux/man-pages/man1/hexdump.1.html>.
+
+- Display the input offset in hexadecimal, followed by three columns of data in octal, per line:
+
+`hexdump -b {{path/to/file}}`
```

Comment:
```suggestion
- Display the input offset in hexadecimal, followed by three columns of data in octal, per line:

`hexdump --one-byte-octal {{path/to/file}}`
```

Mandatory fix: [long option must be used if it's portable](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax).